lib/cylc/batch_sys_handlers/slurm.py

```diff
@@ -74,5 +74,9 @@ def get_fail_signals(self, job_conf):
         """
         return ["EXIT", "ERR", "XCPU"]
 
```

Comment:
(Note: This is not really a forgotten style problem. The reason why it looks so different is because `squeue` has a very different syntax for specifying multiple job IDs. I'll have a look to see whether it is possible to improve the style or not.)